doc/user-guides/kernel-parameters.rst

```diff
@@ -201,11 +201,27 @@ relevant for configuring or debugging ACRN-based systems.
 
    * - hvlog
      - Service VM
-     - Reserve memory for the ACRN hypervisor log. The reserved space should not
-       overlap any other blocks (e.g.  hypervisor's reserved space).
-     - ::
-
-          hvlog=2M@0x6de00000
+     - Sets the guest physical address and size of the ring buffer between
+       hypervisor and SOS dedicated for hypervisor log. It is required that a
+       ``memmap`` parameter is present as well to reserve the specified memory
+       from the guest OS.
+       
+       If hypervisor relocation is disabled, need to check against
+       CONFIG_HV_RAM_START and CONFIG_HV_RAM_SIZE to ensure that the buffer
+       is not overlap with hypervisor's reserved space, provided that in SOS,
+       GPA and HPA is 1:1 mapping.
+
+       If hypervisor relocation is enabled, must reserve the memory below
+       256MB, since hypervisor could be relocated anywhere between 256MB and
+       4GB.
+       
+       Also it is recommended to enable ASLR on SOS. This is to ensure
```

Comment:
change ``Also it is recommended to enable`` to ``You should enable``
change ``This is to ensure that`` to ``This ensures``

Reply:
@dbkinder Thank you very much for the fixes! I have updated the PR.